test/io/jesi/backpack/compare_test.cljc

```diff
@@ -48,8 +48,14 @@
       (are [args] (compares? args)
         [nil nil]
         [0 0]
+        [1 1 1]
         #?@(:clj [(let [now (Instant/now)]
-                    [now now])])))))
+                    [now now])])))
+
+    (testing "same for unordered"
+      (are [args] (compares? args)
+        [1 2 1]
+        ["A" "B" "A"]))))
```

Comment:
I think it would be good to have a test for 3 different values, i.e. [1 2 3]

also an even number of values (greater than 2) i.e. [1 2 3 4]

Reply:
> I think it would be good to have a test for 3 different values, i.e. [1 2 3]
Updated

Reply:
> also an even number of values (greater than 2) i.e. [1 2 3 4]

That is already covered in in the ascending and descending tests. see `colls` value